### ISSUE TITLE
Ingest Hausumringe Bavaria

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#145]
## User story
1. As a User
2. I want to see the building borders (Hausumringe, 2d polygon) for certain lots in Bavaria
3. So that I can display them

## Acceptance criteria
* Ingest and access: https://geodaten.bayern.de/opengeodata/OpenDataDetail.html?pn=hausumringe
* #146 
* #151 
* #147 
* #206 
* #283 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
